src/types/classgroup.py

```diff
@@ -1,15 +1,26 @@
 from dataclasses import dataclass
 
-from src.util.ints import int512
+from src.consensus.constants import ConsensusConstants
 from src.util.streamable import Streamable, streamable
 
 
 @dataclass(frozen=True)
 @streamable
 class ClassgroupElement(Streamable):
-    a: int512
-    b: int512
+    data: bytes
 
     @staticmethod
     def get_default_element():
-        return ClassgroupElement(int512(2), int512(1))
+        # Bit 3 in the first byte of serialized compressed form indicates if
+        # it's the default generator element.
+        return ClassgroupElement(b"\x08")
+
+    @staticmethod
```

Comment:
This should not be in this class. Also, the default element is not a bad element